$DragonFly: src/gnu/usr.bin/groff/tmac/patches/eqnrc.no_obj.patch,v 1.1 2005/09/01 20:47:14 asmodai Exp $

--- eqnrc	2005-08-31 14:28:07.000000000 +0200
+++ eqnrc	2005-08-31 15:42:52.000000000 +0200
@@ -55,6 +53,7 @@
 ifdef latin1 ! define n %1% !
 ifdef utf8 ! define n %1% !
 ifdef cp1047 ! define n %1% !
+ifdef koi8-r ! define n %1% !
 ifdef n !
 set nroff 1
 !
